src/tblib/__init__.py

```diff
@@ -19,22 +19,16 @@
 
 
 class _AttrDict(dict):
+    __slots__ = ()
+
     def __getattr__(self, attr):
         return self[attr]
 
 
-class __traceback_maker(Exception):
```

Comment:
No real problem, I'll go ahead and add it back in as the debugging thing is actually a good point.  Checking the exception type could have been useful in debugging this very issue (and was how I noticed the nlocals thing), so I'll put it back.